src/OpenSpaceToolkit/Astrodynamics/Flight/System/SatelliteSystem.cpp

```diff
@@ -144,6 +144,22 @@ SatelliteSystem SatelliteSystem::Undefined()
     };
 }
 
+SatelliteSystem SatelliteSystem::Default()
+{
+    const Composite satelliteGeometry(Cuboid(
+        {0.0, 0.0, 0.0}, {Vector3d {1.0, 0.0, 0.0}, Vector3d {0.0, 1.0, 0.0}, Vector3d {0.0, 0.0, 1.0}}, {1.0, 2.0, 3.0}
+    ));
+
+    return {
+        Mass(1000.0, Mass::Unit::Kilogram),
+        satelliteGeometry,
+        Matrix3d::Identity(),
+        1.0,
+        2.1,
```

Comment:
I'm nit-picking here because I've been thinking lately (and I think we spoke about this as well) about how defaulting the propagator with sensible number to help get a user off the ground before they start caring about building a satellite system.